Java/ds/README.md

```diff
@@ -1 +1,2 @@
-a
+
+*  [Sparse Matrix](SparseMatrix.java)
```

Comment:
Don't make the README updates here

Reply:
I have deletd that and updated the correct readme.md file